HuskStandalone/HuskStandalone.param

```diff
@@ -23,4 +23,22 @@ CategoryOrder=0
 Index=0
 Label=HUSK Executable
 Description=The path to the husk executable file used for rendering. Enter alternative paths on separate lines.
-Default=C:\Program Files\Side Effects Software\Houdini 18.5.596\bin\husk.exe
+Default=C:\Program Files\Side Effects Software\Houdini 19.5.000\bin\husk.exe
+
+[USD_RenderExecutable_19_5]
+Type=multilinemultifilename
+Category=Render Executables
+CategoryOrder=0
+Index=0
+Label=HUSK 19.5 Executable
+Description=The path to the husk executable file used for rendering. Enter alternative paths on separate lines.
+Default=C:\Program Files\Side Effects Software\Houdini 19.5.000\bin\husk.exe
+
+[USD_RenderExecutable_20_0]
+Type=multilinemultifilename
+Category=Render Executables
+CategoryOrder=0
+Index=0

```

Comment:
Should one of these indexes be non-zero?

Reply:
tbh, I don't know. 
I think Deadline can handle it.


 
> Plugin Configuration ![image](https://private-user-images.githubusercontent.com/20871534/334180472-727a5aab-6452-485b-8f1d-4601331914c7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY4ODU2NTMsIm5iZiI6MTcxNjg4NTM1MywicGF0aCI6Ii8yMDg3MTUzNC8zMzQxODA0NzItNzI3YTVhYWItNjQ1Mi00ODViLThmMWQtNDYwMTMzMTkxNGM3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTI4VDA4MzU1M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFmMmRlOWVhNTc3ZDZhNjBlY2VkMzU2OWJhMGEwNjg3M2MxZDlhYjA5NWE0MTNlNGZhOTQwNWEwYTg1ZDZlMzcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.RkN4YCbBpDvrn_w-bJl8nM_BSWIHJ4w-m38as8HRxME)
> 
> Job Properties ![image](https://private-user-images.githubusercontent.com/20871534/334180520-816679cb-f4cd-4b8c-ba72-a92089786193.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY4ODU2NTMsIm5iZiI6MTcxNjg4NTM1MywicGF0aCI6Ii8yMDg3MTUzNC8zMzQxODA1MjAtODE2Njc5Y2ItZjRjZC00YjhjLWJhNzItYTkyMDg5Nzg2MTkzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTI4VDA4MzU1M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY4NWU1MTE5ZDcxOWQwNjc0NGJmN2MxOTU0OTJiMjM2MjNkY2U1MzUwOTI4ZmM1NmMxODMyOTg5MmVjYTdkOTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Ps31_r6-JLpwIftCZ-bhym0n3_oJdwyYRkVQE5aB6F0)

Reply:
Images don't seem to work, anyway. Looking at Deadline's only `.param` file for e.g. for `MayaBatch` the indexes do increase. So let's just match it to be sure.

```

[RenderExecutable2010_0]
Type=multilinemultifilename
Category=Render Executables
CategoryOrder=0
Index=0
Label=Maya 2010 Render Executable
Default=C:\Program Files\Autodesk\Maya2010\bin\MayaBatch.exe;C:\Program Files (x86)\Autodesk\Maya2010\bin\MayaBatch.exe;/usr/autodesk/maya2010-x64/bin/maya;/Applications/Autodesk/maya2010/Maya.app/Contents/bin/maya
Description=The path to the Maya 2010 executable file used for rendering. Enter alternative paths on separate lines.

[RenderExecutable2011_0]
Type=multilinemultifilename
Category=Render Executables
CategoryOrder=0
Index=1
Label=Maya 2011 Render Executable
Default=C:\Program Files\Autodesk\Maya2011\bin\MayaBatch.exe;C:\Program Files (x86)\Autodesk\Maya2011\bin\MayaBatch.exe;/usr/autodesk/maya2011-x64/bin/maya;/Applications/Autodesk/maya2011/Maya.app/Contents/bin/maya
Description=The path to the Maya 2011 executable file used for rendering. Enter alternative paths on separate lines.

[RenderExecutable2011_5]
Type=multilinemultifilename
Category=Render Executables
CategoryOrder=0
Index=2
Label=Maya 2011.5 Render Executable
Default=C:\Program Files\Autodesk\Maya2011.5\bin\MayaBatch.exe;C:\Program Files (x86)\Autodesk\Maya2011.5\bin\MayaBatch.exe;/usr/autodesk/maya2011.5-x64/bin/maya;/Applications/Autodesk/maya2011.5/Maya.app/Contents/bin/maya
Description=The path to the Maya 2011.5 executable file used for rendering. Enter alternative paths on separate lines.
```

Which would make this
```suggestion
CategoryOrder=0
Index=0
Label=HUSK 19.5 Executable
Description=The path to the husk executable file used for rendering. Enter alternative paths on separate lines.
Default=C:\Program Files\Side Effects Software\Houdini 19.5.000\bin\husk.exe

[USD_RenderExecutable_20_0]
Type=multilinemultifilename
Category=Render Executables
CategoryOrder=0
Index=1

```

Reply:
https://github.com/BigRoy/HuskStandaloneSubmitter/pull/2/commits/ba4e08eb2cf4e156828e301e549e1e77553962bd